### DISCUSSION TITLE
Looking for Developer to Create Simple Software for Hikvision DS-TCG406-E ANPR Camera

### DISCUSSION THREAD

**Discussion by vlasov7979** [community/community#143378] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Dear Forum Members! Greetings to all!

Please, go easy on a newbie here. I need some help (don’t worry, it’s a paid gig!) with creating software for the Hikvision DS-TCG406-E camera. The main functions involve accessing the video stream, controlling the relay, managing black and white lists, and displaying transit statistics (time and license plate number). Preferred development languages are Delphi or C++. A simple interface will do just fine!

**Comment by gosha-z** [comment]
Hello!

   Looks interesting. If your request is still actual - how can I contact you? Telegram preferred

Regards
I.

**Comment by vlasov7979** [comment]
Hello Dear Friend. Telegram @SVGtechnology

**Comment by LiteBrite82** [comment]
Hi @vlasov7979 , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
